src/utils/image.js

```diff
@@ -16,31 +21,41 @@ function getImage(body, extension) {
   }
 }
 
-export const fetchImage = src => {
-  if (typeof src === 'object') {
-    if (src.data && src.format) {
-      // Local file given
-      return new Promise((resolve, reject) =>
-        resolve(getImage(src.data, src.format)),
-      );
-    }
-    throw new Error(
-      `Invalid data given for local file: ${JSON.stringify(src)}`,
-    );
+const resolveBase64Image = src => {
+  const match = /data:image\/([a-zA-Z]*);base64,([^\"]*)/g.exec(src);
```

Comment:
This regex is identical to the `isBase64` regex besides the catching groups, right? If so, they should be able to use the same, extracted to a single `const`, just for simplicity.